src/main/resources/flyway/sql/V10__populate_institutions.sql

```diff
@@ -0,0 +1,443 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+-- Delete two dummy institutions --
+DELETE FROM program_institution WHERE institution_id IN (SELECT id FROM institution WHERE NAME = 'OICR');
```

Comment:
You should insert the new institutions before deleting existing ones and then migrate existing associations to those. 

The way it is right now, any existing testing programs or otherwise will not have institutions, thus putting them in an inconsistent state.